### PR TITLE
Added some feature

### PR DESCRIPTION
**Pull request by 2pai**
1. update eslint rule
2. add basic auth
3. add logger
4. add lint:fix command
5. improve the code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "serve": "nodemon server.js"
+    "serve": "nodemon server.js",
+    "lint:fix": "./node_modules/.bin/eslint src --fix"
```
Comment (BlinfoldKing) [inline package.json]: lint nya di prestart ae, trus serve nya ganti ke start jadi tiap kali mau start server dia langsung fix lint nya btw jaga perlu `./node_modules/.bin/eslint` udah auto nyari di local

Comment (BlinfoldKing) [reply package.json]: sama kalo pengen lebih yakin lagi pasang githook setiap commit, jadi kaga bisa commit kalo eslintnya masih error

File: .env.example [hunk 2]
```diff
@@ -1 +1,3 @@
-PORT=3022
\ No newline at end of file
+PORT=3022
+AUTH_BASIC_USERNAME = 'admin'
+AUTH_BASIC_PASSWORD = 'fc9a2d1e-a12e-5b45-86b3-94bebcdea60d'
```
Comment (BlinfoldKing) [inline .env.example]: mau pake spasi atau kaga nih, biasanya sih kaga pake spasi, dan kaga perlu pake kutip

File: package.json (hunk 1 above)
Comment (2pai) [reply package.json]: dibikin gini mas ? @BlinfoldKing 
```
    "start": "nodemon server.js",
    "prestart": "npm run lint",
    "lint:fix": "eslint src --fix"
```

File: .env.example (hunk 2 above)
Comment (2pai) [reply .env.example]: fixed https://github.com/2pai/kiana/pull/2/commits/cfe2a004bdd2aacbe35be97c2b865f3b02f82be0

File: package.json (hunk 1 above)
Comment (2pai) [reply package.json]: addressed here https://github.com/2pai/kiana/pull/2/commits/53d875823e80a4f808389b9ebf07ebe3392ff6c5

**Comment by BlinfoldKing** [comment]
lgtm

**Comment by BlinfoldKing** [comment]
kalo merge di squash yakn

**Pull request merged**